assets/files/etc/NetworkManager/dispatcher.d/pre-up.d/z-baremetalupdatescript

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+IF=ens4
+BRIDGE=baremetal
+if [ "$1" == "$IF" ] && [ -z "$( ip a | grep $IF | grep $BRIDGE)" ] ; then
+ cp -f /etc/resolv.conf /etc/resolv.conf.ori
+ DEFAULT_ROUTE=$( ip r | grep default | grep $IF )
+ for ip in $(ip -o addr |grep $IF | grep -v inet6 | awk '{print $4}') ; do
```

Comment:
filtering out ipv6 addressing that was put there by the dc dhcp seems a bit evil

Reply:
changed the logic